### PR TITLE
Feature/get current user

### PR DESCRIPTION
**Pull request by cesar-oyarzun-m**

### CODE REVIEW

File: exp-android-sdk/src/main/java/com/scala/exp/android/sdk/Runtime.java [hunk 1]
```diff
@@ -166,6 +171,75 @@ public void call(Throwable throwable) {
                 });
     }
 
+
+    /**
+     * Start SDK with Auth
+     * @param auth
+     * @return
+     */
+    public static Observable<Boolean> start_auth(final Auth auth){
```
Comment (adamgallowayscala) [inline exp-android-sdk/src/main/java/com/scala/exp/android/sdk/Runtime.java]: use camel case for method names

File: exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Auth.java [hunk 2]
```diff
@@ -78,4 +80,9 @@ public Network getNetwork() {
     public void setNetwork(Network network) {
         this.network = network;
     }
+
+    public Api getApi() {return api;}
```
Comment (adamgallowayscala) [inline exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Auth.java]: formate on new line, like other methods

File: exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Auth.java [hunk 3]
```diff
@@ -78,4 +80,9 @@ public Network getNetwork() {
...
+
+    public void setApi(Api api) {this.api = api;}
```
Comment (adamgallowayscala) [inline exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Auth.java]: format on new line

File: exp-android-sdk/src/main/java/com/scala/exp/android/sdk/Runtime.java (hunk 1 above)
Comment (adamgallowayscala) [reply exp-android-sdk/src/main/java/com/scala/exp/android/sdk/Runtime.java]: Try to combine this into the other start_auth() method, they seem to share a lot of code

**Review by adamgallowayscala — CHANGES_REQUESTED** [review]

**Comment by cesar-oyarzun-m** [comment]
@adamgallowayscala PR fixed!

**Review by adamgallowayscala — APPROVED** [review]

**Pull request merged**